packages/mobile/src/home/reducers.ts

```diff
@@ -53,11 +56,17 @@ export const homeReducer = (state: State = initialState, action: ActionTypes | R
         if (!updatedNotification) {
           continue
         }
+        const existingNotification = state.notifications[id]
         updatedNotifications = {
           ...updatedNotifications,
           [id]: {
-            ...(state.notifications[id] || {}),
             ...updatedNotification,
+            // Keep locally modified fields
```

Comment:
This only affects us if we put a value for `dismissed` in Firebase, right? If there's no such field on Firebase, we were still keeping the locally stored value.

If that's the case I'm not sure I see the need for this change, but I don't see anything wrong with it either. Might be missing something though.

Reply:
Yes what I saw happening while testing is that previously set values would remain in the redux store.
For instance if I set `minVersion` to something for testing and then would delete the field in Firebase, it would stay in the redux state. Leading to the notification remaining visible hidden or visible when it shouldn't according to what's in Firebase.